packages/catalog-process/src/services/catalogService.ts

```diff
@@ -1194,16 +1281,25 @@ export function activateDescriptorLogic({
     recentDescriptorVersion !== null &&
     parseInt(descriptor.version, 10) === recentDescriptorVersion
```

Comment:
`parseInt` should be no more required for `version` field.
May I ask you to check also other occurrences?

Also, can it be now changed to this?
```suggestion
    descriptor.version === recentDescriptorVersion
```

Reply:
it's a good point, thanks.
Let's leave the readmodel model with the string type, I'll create a task to change it once the migration will be completed